zenith_utils/src/pq_proto.rs

```diff
@@ -768,3 +770,14 @@ impl<'a> BeMessage<'a> {
         Ok(())
     }
 }
+
+// Zenith extension of postgres replication protocol
+// See ZENITH_STATUS_UPDATE_TAG_BYTE
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ZenithStatusUpdate {
+    // The last LSN that is synced to remote storage and is guaranteed to survive pageserver crash
+    // Used by safekeepers to remove WAL preceding `remote_consistent_lsn`.
+    pub remote_consistent_lsn: Lsn,
```

Comment:
I still don't get why can't we put lsn in a usual feedback message to keep them together. IMO it would make more sense to move backpressure-related lsn here since it is semantically different from usual ones